orderable/tests/test_models.py

```diff
@@ -207,3 +213,13 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
```

Comment:
Found the bugs how?  It looks like it improved on the hardcoded list in the test that was there already, but wouldn't changing that to `[1,2,3,4]` as you described earlier have achieved the same thing?  Did you get that from the test output?  Do you think you would have been able to find this error using Hypothesis without that test already in place (or without writing it yourself)?

And yes - the point being that there probably aren't many more edge cases or failures in the exact list of `sort_orders` you have (there are only so many different relevant situations you can construct), but there might be plenty of edge cases or failures elsewhere that this test will never hit.  Adding Hypothesis here doesn't guarantee much.